addons/tutorial_01_get_started/manifest.json

```diff
@@ -17,6 +14,7 @@
       {
         "id": "mainScreen",
         "element": "serverListButton-btn",
```

Comment:
Shouldn't `element` be removed?

Reply:
no, because this tutorial must be back-compatible.